movie_ratings.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
Looks like your only linking the tables is there any cleaning happening or is the dataset already cleaned?